Doc/communication guidelines (#110)
SuperDARN/pydarn: docs/dev/communication.md

# Communication Guidelines

pyDARN is under the Data Analysis Working Group that has set up guidelines on [communication](https://superdarn.github.io/dawg/documents/communication-guidelines/) to encourage a positive collaborative environment.
Please follow these guidelines when commenting on GitHub. 

Also please read workflow documentation on issues and [pull requests](dev/PR.md) to communicate effectively for collaborative work.

## Disputes Procedure

If members have a dispute that violates the communication guidelines, and creates a negative environment in the pyDARN community, please follow this procedure to resolve the conflict:
1. Please try resolve the matter between each member offline.
2. If the dispute cannot be resolved, or the environment has become toxic and overwhelming, please contact the [lead developer for pyDARN](marina.t.schmidt@gmail.com).

> 3. If the dispute cannot be resolved, involves the lead developer, or breaches the [DAWG charter](https://superdarn.github.io/dawg/reports/202009/charter-update) please contact the [chairs of DAWG](https://www.unis.no/staff/emma-bland/).

Reviewer: Should we link to the "official" version of the charter? I know it still shows the old one on the dawg website, but I'm happy to update it since it's unlikely to change again.
```suggestion
3. If the dispute cannot be resolved, involves the lead developer, or breaches the [DAWG charter](https://superdarn.github.io/dawg/documents/DAWG_Charter/) please contact the [chairs of DAWG](https://www.unis.no/staff/emma-bland/).
```

# Communication Guidelines

pyDARN is under the Data Analysis Working Group that has set up guidelines on [communication](https://superdarn.github.io/dawg/documents/communication-guidelines/) to encourage a positive collaborative environment.
Please follow these guidelines when commenting on GitHub. 

Also please read workflow documentation on issues and [pull requests](dev/PR.md) to communicate effectively for collaborative work.

## Disputes Procedure

If members have a dispute that violates the communication guidelines, and creates a negative environment in the pyDARN community, please follow this procedure to resolve the conflict:
1. Please try resolve the matter between each member offline.
2. If the dispute cannot be resolved, or the environment has become toxic and overwhelming, please contact the [lead developer for pyDARN](marina.t.schmidt@gmail.com).

> 3. If the dispute cannot be resolved, involves the lead developer, or breaches the [DAWG charter](https://superdarn.github.io/dawg/reports/202009/charter-update) please contact the chairs of DAWG.

Author: ``` suggestion
3. If the dispute cannot be resolved, involves the lead developer, or breaches the [DAWG charter](https://superdarn.github.io/dawg/reports/202009/charter-update) please contact the [chairs of DAWG](https://www.unis.no/staff/emma-bland/).
```

@ecbland

# Communication Guidelines

> pyDARN is under the Data Analysis Working Group that has setup some guidelines on [communication](https://superdarn.github.io/dawg/documents/communication-guidelines/) to encourage positive collaborative environment.

Reviewer 2: ```suggestion
pyDARN is under the Data Analysis Working Group that has set up guidelines on [communication](https://superdarn.github.io/dawg/documents/communication-guidelines/) to encourage a positive collaborative environment.
```

Please follow these guidelines when commenting on GitHub. 

> Also please read workflow documentation on issues and [pull requests](dev/PR.md) to communicate properly for collaborative work.

Reviewer 2: ```suggestion
Also please read workflow documentation on issues and [pull requests](dev/PR.md) to communicate effectively for collaborative work.
```
Just a suggestion on wording, I can image someone saying that they are doing it properly, it's just not the way we do it.

## Disputes Procedure

> If one of the members have a dispute that violates the communication guidelines and is creating a negative environment in the pyDARN community, please follow the procedure for resolving the conflict:

Reviewer 2: ```suggestion
If members have a dispute that violates the communication guidelines, and creates a negative environment in the pyDARN community, please follow this procedure to resolve the conflict:
```
Same tense for the whole sentence, but equally could be the opposite tense, I've gone with future.

1. Please try resolve the matter between each member offline.

> 2. If the dispute cannot be resolved or environment has become toxic and overwhelming please contact the [lead developer for pyDARN](marina.t.schmidt@gmail.com).

Reviewer 2: ```suggestion
2. If the dispute cannot be resolved, or the environment has become toxic and overwhelming, please contact the [lead developer for pyDARN](marina.t.schmidt@gmail.com).
```

> 3. If the dispute cannot be resolved, involves the lead developer, or breaches the [DAWG charter](https://superdarn.github.io/dawg/reports/202009/charter-update) please contact the chairs of DAWG.

Reviewer 2: Do we want to add E and K's emails here?

Author: @ksterne and  @ecbland ?

I was not sure so I left it blank
